specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2016-10-01/examples/getSecret.json

```diff
@@ -0,0 +1,28 @@
+{    
+    "parameters": {
+        "resourceGroupName": "sample-group",
+        "vaultName": "sample-vault",
+        "secretName":"secret-name",
+        "subscriptionId": "00000000-0000-0000-0000-000000000000",
+        "api-version": "2016-10-01"
+    },
+    "responses": {
+        "200": {
+            "body": {
+                "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/sample-group/providers/Microsoft.KeyVault/vaults/sample-vault/secrets/secret-name",
+                "name": "secret-name",
+                "type": "Microsoft.KeyVault/vaults/secrets",
+                "location": "westus",
+                "properties": {
+                    "attributes": {
+                        "enabled": true,
+                        "created": 1514940950,
+                        "updated": 1514940950
+                    },
```

Comment:
Missing required property "value".

Reply:
Same for listSecrets.json

Reply:
value is not actually required in this model.  I've update the model to reflect that.